composer.json

```
@@ -11,11 +11,12 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "codeception/codeception": "^3.0"
+        "php": ">=7.2.0",
+        "codeception/codeception": "^4.0"
```

since you never changed any code I presume that it is compatible with codeception 3.0, so ^3.0.0 | ^4.0.0 would be good

---

```suggestion
        "codeception/codeception": "^3.0 || ^4.0"
```

I think this decision is up to @tiger-seo cause it's mainly about the support commitment. It seems to currently work with both versions but this may increase maintenance costs or require another major release later. Added it as a suggestion so that tiger-seo can decide to apply or not before merging.

---

well, if they break compatibility, then according to SEMVER they have to bump major, but since major is locked from version 3.x.x to 4.x.x then it should not be a problem